### PR TITLE
Loginurl attribute fix for project create

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Adds the missing login url attribute to project create. I'm still working on the tests but want to get an early review on the fix.

### CODE REVIEW

File: src/commands/force/project/create.ts [hunk 1]
```diff
@@ -52,6 +52,12 @@ export default class Project extends TemplateCommand {
     manifest: flags.boolean({
       char: 'x',
       description: messages.getMessage('manifest')
+    }),
+    loginurl: flags.string({
+      char: 'l',
```
Comment (lcampos) [inline src/commands/force/project/create.ts]: I'm actually not sure about the character value, anyone knows if this is correct ?

Comment (AnanyaJha) [reply src/commands/force/project/create.ts]: Tested it with l and it works!

Comment (lcampos) [reply src/commands/force/project/create.ts]: I've confirmed that is the short value for the attribute in the old repo.

File: src/commands/force/project/create.ts [hunk 2]
```diff
@@ -62,7 +68,9 @@ export default class Project extends TemplateCommand {
     this.flags.ns = this.flags.namespace;
 
     // TODO: update the latest apiversion
-    this.flags.sourceApiVersion = '46.0';
+    this.flags.sourceApiVersion = '47.0';
```
[the remark this answers is not in the document]
Comment (lcampos) [reply src/commands/force/project/create.ts]: Agreed, I'll open a bug for that

[the remark this answers is not in the document]
Comment (lcampos) [reply src/commands/force/project/create.ts]: Done, work item is W-6825719

**Pull request merged**